src/Stratis.Bitcoin.Features.Wallet/WalletTransactionHandler.cs

```diff
@@ -291,6 +291,7 @@ protected void AddCoins(TransactionBuildContext context)
             if (balance < totalToSend)
                 throw new WalletException("Not enough funds.");
 
+            Money sum = 0;
```

Comment:
There was a bug here in the previous code, you need to account for the fee, I suggest adding the max fee as starting point.

Reply:
OK, I see let me log issue for your comment. Thanks